### PR TITLE
Consolidating oak_functions protos under the top level proto package

### PR DESCRIPTION
**Pull request by k-naliuka**

### CODE REVIEW

**Comment by tiziano88** [comment]
Thanks for starting on this! BTW we still want to keep the full `oak_functions` name in the path (i.e. not just `functions`).

**Comment by k-naliuka** [comment]
> Thanks for starting on this! BTW we still want to keep the full `oak_functions` name in the path (i.e. not just `functions`).

Thanks! Renamed the package

File: proto/functions/abi.proto
Comment (tiziano88) [inline proto/functions/abi.proto:1]: It looks like there are multiple copies of these files now? Or am I looking at the wrong diff?

**Review by tiziano88 — COMMENTED** [review]
Note that as soon as this is submitted, copybara will apply the corresponding changes in google3, which will require manual changes. So please submit when you think you have a window of time to also fix the google3 code, otherwise it will stay in a broken state for too long.

**Comment by tiziano88** [comment]
BTW going forward, please don't push directly to the oak repo, but create a fork on your account and develop there

File: proto/functions/abi.proto
Comment (k-naliuka) [reply proto/functions/abi.proto:1]: You're right, I somehow managed to restore the deleted ones with Git :-/ Removed them

**Review by tiziano88 — APPROVED** [review]
Cool, good start! Thanks

**Comment by tiziano88** [comment]
Note that there are a lot of spurious changes at the moment in this PR. Not sure they will be cleanly resolved by squashing, as they seem to replicate existing commits that were already merged.

**Pull request merged**